sources/packages/backend/apps/api/src/services/application-bulk-withdrawal/application-bulk-withdrawal-import-validation.service.ts

```diff
@@ -0,0 +1,149 @@
+import { BadRequestException, Injectable } from "@nestjs/common";
+import {
+  ValidationResult,
+  ValidationContext,
+  ValidationContextTypes,
+  ApplicationBulkWithdrawalImportBusinessValidationModel,
+  ApplicationWithdrawalValidationResult,
+} from "./application-bulk-withdrawal-import-business-validation.models";
+import { validateSync, ValidationError } from "class-validator";
+import { plainToClass } from "class-transformer";
+import { flattenErrors } from "../../utilities/class-validation";
+import { CustomNamedError } from "@sims/utilities";
+import {
+  APPLICATION_BULK_WITHDRAWAL_VALIDATION_CRITICAL_ERROR,
+  APPLICATION_WITHDRAWAL_VALIDATION_ERROR,
+} from "../../constants";
+import { ApiProcessError } from "../../types";
+import { ApplicationBulkWithdrawalValidationResultAPIOutDTO } from "../../route-controllers/student-scholastic-standings/models/student-scholastic-standings.dto";
+
+/**
+ * Executes the validations on an offering model.
+ */
+@Injectable()
+export class ApplicationBulkWithdrawalImportValidationService {
+  /**
+   * Validate offering models and provide the result for every model.
+   * @param businessValidationModels application bulk withdrawal business validation models to be validated.
+   * @param silently if true, no exception is generated case the validation fails
+   * and the success or failure can be determined from the result object.
+   * @returns validation result or an exception in the case of a failed validation
+   * combined with the silently parameter defined as false.
+   */
+  validateApplicationBulkWithdrawalModels(
+    businessValidationModels: ApplicationBulkWithdrawalImportBusinessValidationModel[],
```

Comment:
If `business` is removed everywhere I believe that it would not reduce the meaning.